test/performance/README.md

```diff
@@ -0,0 +1,82 @@
+# Kueue performance testing
+
+## Measurements
+
+### Job startup latency
+
+How fast do jobs transition from `created` to `started` state?
+Time spent between the transition from `job.CreationTimestamp.Time` to `job.Status.StartTime.Time` state.
+
+### Job startup throughput
+
+Workload admission [rate per second] over minute intervals, Prometheus query:
+
+[To be fine-tuned]
+`max_over_time(sum(rate(kueue_admitted_workloads_total{cluster_queue="{{$clusterQueue}}"}[1m]))[{{$testTimeout}}:5s])`
+
+## How to run the test?
+
+### Prerequisites
+
+1. Deploy [Kueue](https://github.com/kubernetes-sigs/kueue/blob/main/docs/setup/install.md)
+2. Make sure you have `kubectl`, `jq`, [golang version](https://github.com/mikefarah/yq) of `yq` and `go` 
```

Comment:
link to `jq` installation.

Although we generally have all dependencies installed through the Makefile.

Reply:
Added the link